packages/typespec-vscode/src/types.ts

```diff
@@ -1,6 +1,7 @@
 export const enum SettingName {
   TspServerPath = "typespec.tsp-server.path",
   InitTemplatesUrls = "typespec.initTemplatesUrls",
+  Emitters = "typespec.emitters",
```

Comment:
how about typespec.emitCode.emitters? typespec.emitters is too big, we may add more features that needs to configure emitters too in the future which would then make typespec.emitters confusing.

Reply:
We will use `kind` to indicate the type of emitter (client, server, schema or others), so we will not separate emitters according to the type in settings.

Reply:
changed to `GenerateCodeEmitters = "typespec.generateCode.emittters"`